Fix for NanoHub Ticket #378563

JackS9 opened the pull request:
This adds a help entry for adding an external link to a course outline. Fixes NanoHub Ticket #378563.

JackS9 committed:
Fix for NanoHub Ticket #378563

nkissebe commented:
Jack, can you change the example to use a URL like example.com

zooley commented on core/components/com_courses/site/help/en-GB/builder.phtml:
```diff
@@ -22,4 +22,13 @@ defined('_HZEXEC_') or die();
\ No newline at end of file
+</pre>
+
+<a name="link"></a><h2>Embedding an exteral link into your outline</h2>
```
Minor note: The `name` attribute on anchors isn't valid in HTML 5. The preferred method is to use an `id`.

```
<h2 id="link"> ...
```

There's undoubtedly a number of these throughout the core that need to get updated.

dbenham committed:
Update builder.phtml

made mods myself, they were easier to make than request another person to do them.

dbenham reviewed (commented):
I changed name to id, and used a generic example.com link in the example.

dbenham commented:
merging change to dev, this is not a critical fix, so no hot fix will be created. But going to review code deployment procedures, not specifically for this fix, but fixes in general.
